.travis.yml

```diff
@@ -0,0 +1,76 @@
+arch: arm64-graviton2
+dist: focal
+virt: vm
+group: edge
+language: python
+services: docker
+env:
+  global:
+      - REPO_DIR=pandas
+      # on release change this to a version
+      - BUILD_COMMIT="v1.1.2"
+      - UNICODE_WIDTH=32
+      - PLAT=aarch64
+      - NP_BUILD_DEP="numpy==1.13.3"
+      - NP_TEST_DEP="numpy==1.13.3"
+      - CYTHON_BUILD_DEP="cython==0.29.16"
+      - WHEELHOUSE_UPLOADER_USERNAME=travis-worker
+      - TEST_DEPENDS="pytest pytest-xdist hypothesis"
+      - TEST_DIR="tmp_for_test"
+      - MB_ML_VER=2014
+      - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+      # Following generated with
+      # travis encrypt -r MacPython/pandas-wheels WHEELHOUSE_UPLOADER_SECRET=<the api key>
```

Comment:
Has this secret been uploaded? I don't see it in https://travis-ci.com/github/MacPython/pandas-wheels/settings, but I'm not sure if I should be seeing it.

Reply:
You are correct. The secret is wrong anyway, it should be the secret(s) from anaconda.org, like the [azure pipeline](https://github.com/MacPython/pandas-wheels/blob/master/azure/posix.yml#L89).

Reply:
So someone will need to add those tokens to travis-ci.com

Reply:
I added `PANDAS_STAGING_UPLOAD_TOKEN` and `PANDAS_NIGHTLY_UPLOAD_TOKEN` through the travis UI.